mvg/mvg.py

```diff
@@ -46,8 +46,11 @@ def __init__(self, endpoint: str, token: str):
         self.endpoint = endpoint
         self.token = token
 
-        self.mvg_version = self.parse_version("v0.4.1")
-        self.tested_api_version = self.parse_version("v0.1.0")
+        self.mvg_version = self.parse_version("v0.5.0")
```

Comment:
This is more of a personal question, but had we defined somewhere when a change deserves to upgrade version in the last decimal, the middle one or the first one?

Reply:
This is following semantic versioning convention (semver). It is explained [here](https://semver.org/)

Reply:
Thanks a lot for the answer